packages/odyssey-react/src/components/Box/_box-border.scss

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+$outer-border-radius: 6px;
```

Comment:
Is there a reason this border-radius is 6px? Our border-radius is 4px

Reply:
The base border radius remains 4px
https://github.com/okta/odyssey/pull/1157/files#diff-26904dc5f87a482c686badf79d203f2aae84dd8301e0422a9aa1c1714bd9a808R37

I added this 6px value based on the designs from Jose which proposed an addition value for wider outer borders.

Reply:
Could we add this to our `tokens.scss` instead if we want it to be considered a global token?

I am worried it will get lost in the shuffle if it's not co-located with others.